docker/app/pyramid-settings.ini.jinja

```diff
@@ -47,7 +50,7 @@ frontend.petition.urls.url = {{ context['frontend']['domain'] }}/petitions/{id}
 frontend.petition.urls.edit = {{ context['frontend']['domain'] }}/petitions/{id}/edit
 frontend.petition.urls.city = {{ context['frontend']['domain'] }}/petitions/{city_url_id}
 frontend.petition.urls.letter = https://{{ context['access']['url'] }}/#/petition/{id}/letter/cover
-# frontend.petition.urls.appendix = {{ context['frontend']['domain'] }}/petitions/{id}/appendix
+frontend.petition.urls.appendix = https://{{ context['access']['url'] }}/#/petition/{id}/letter/appendix?apitoken={apitoken}
```

Comment:
Here it is called apitoken at other places it is called apikey.

I would say it is just a `token`. Please change this everywhere.